res/META-INF/plugin.xml

```diff
@@ -71,7 +71,7 @@
 	<actions>
 		<action
 				class="org.ziglang.action.NewZigFile"
-				icon="ZigIcons.ZIG_FILE"
+				icon="/icons/zig_file.png"
```

Comment:
I do not think this is the best solution.

But I was not able to make kotlin class work here. For some reason plugin loader is unable to finds `ZigIcons.ZIG_FILE` here. It feels like files generated by kotlin are in different class loader than the java one.

cc @ice1000 @linux-china for a better solution

Reply:
I think you should keep the original code.